javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,219 @@
+
+let nameArray = [];
+let todoArray = [];
+
+function getReply(command)
+{
+    let wordString = command.split(" "); 
+    let KeyWord = wordString[0];  // extract the keyword
+    let nameString;
+    let nameString2;
+
+    switch(KeyWord){
+        //--------------------------------------------------------------//
+        case 'Hello':
+            nameString = wordString[4]; // extract the name (as it is on 4th index)
+            if(nameString == '')
```

Comment:
Empty strings evaluate to false, so this could be written like:
`if(!nameString)`